core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -893,6 +893,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_BACKUP_DIRECTORY =
+      new Builder(Name.MASTER_BACKUP_DIRECTORY)
+          .setDefaultValue("/alluxio_backups")
+          .setDescription("Default directory for writing master metadata backups.")
```

Comment:
This is the directory for which FS? root mount? local? If for root UFS, is it the directory relative to the  mount point, or relative to root?

Reply:
updated to
```java
          .setDescription("Default directory for writing master metadata backups. This path is"
              + " relative to the root directory of the root UFS. For example, if the root ufs"
              + " directory is hdfs://cluster/alluxio/data, the default backup directory will be"
              + " hdfs://cluster/alluxio_backups")
```